autoload/gitgutter.vim

```diff
@@ -191,6 +191,9 @@ function! gitgutter#quickfix()
         \ ' diff --no-ext-diff --no-color -U0'.
         \ ' --src-prefix=a/'.path_to_repo.' --dst-prefix=b/'.path_to_repo.' '.
         \ g:gitgutter_diff_args. ' '. g:gitgutter_diff_base
+  if g:gitgutter_use_location_list
+    let cmd = cmd.expand(expand('%:p'))
```

Comment:
Why two `expand()`s?

Also I think it would be more robust to prefix the path with `--`.

Reply:
huh thats odd. probably a typo. 

+1 on `--`